src/hw-2/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface NewUser {
```

Comment:
Logically I don't see a reason to separate a type User given in the hometask

Reply:
Removed.